tests/languages/powerquery/number_feature.test

```diff
@@ -0,0 +1,43 @@
+42
+-42
++42
+0.154
+.123
+0xF2
+42e5
+42.5e2
+123
+123e45
+123e+45
+123e-45
+123E45
+123E+45
+123E-45
+123.78e-45
+.78e-45
+

```

Comment:
Could you please add an example for a range like `123..456`. This broke the pattern before, so I think we should include a test case.

Reply:
OK, I have added it here to check that it properly recognizes the `..` operator

Reply:
Should I simplify the testfile for the functions, maybe?

Reply:
Yeah, that'd be nice. Maybe also add a case for user-defined functions.

After that, I think this PR is good to go.

Reply:
OK done. Function tests are now very simple, hope this is OK.